locales/ro.json

```diff
@@ -1127,5 +1135,93 @@
+      "p1": "Poți alege oricare din metodele de mai jos:",
+      "list": [
+        "Descarcă formularul și completează-l digital folosind aplicația Adobe Reader, conform video-ului de mai jos. Printează apoi formularul și semnează-l. ",
+        "Descarca formularul, printează-l, completează-l de mâna și semnează-l."
+      ],
+      "p2": "Aici găsesti instrucțiunile de completare precum și un model de formular completat. "
+    },
+    "faq": {
+      "q1": {
+        "question": "Trebuie să calculez eu cât înseamnă cota de 3,5% și să scriu în Cerere?",
+        "answer": "Nu trebuie să scrii nimic în câmpul respectiv. Sumă va fi calculată și virată de organul fiscal în funcție de venitul declarat."
+      },
+      "q2": {
+        "question": "Am completat deja o cerere pentru un ONG/unitate de cult însă m-am răzgândit. Pot să schimb opțiunea?",
+        "answer": "Da. Poți completa un alt formular. Dacă formularele nu sunt depuse personal de către tine, ANAF îți va trimite o notificare prin care îți va cere să confirmi alegerea. Această procedură se realizează pentru combaterea fraudelor."
+      },
+      "q3": {
+        "question": "Ce trebuie să fac dacă primesc notificare de la ANAF cu privire la redirecționarea celor 3.5% din impozit?",
+        "answer": [
+          "ANAF trimite scrisori tuturor contribuabililor care nu depun personal formularul 230.",
+          "Verifică datele beneficiarului trecut pe cererea depusă în numele tău iar dacă acestea corespund opțiunii tale nu trebuie să faci nimic.",
+          "Dacă există neregului și nu ai optat pentru acel beneficiar, îți recomandăm să te adresezi unității fiscal deoarece poate fi vorba despre o fraudă, cineva a completat formularul în numele tău.",
+          "Dacă ai completat un formular iar apoi te-ai razgandit și ai mai depus unul, prin scrisoarea de la ANAF ți se va cere să confirmi opțiunea finală."
+        ]
+      },
+      "q4": {
+        "question": "E obligatoriu să bifez căsuța “Opțiune privind distribuirea sumei pentru o perioadă de 2 ani”?",
+        "answer": "Nu. Dacă dorești ca acest formular să aibă valabilitate doar pentru un an, nu bifa căsuța."
+      },
+      "q5": {
+        "question": "Este corect trecut anul 2020?",
+        "answer": "Da. Formularul se aplică pentru veniturile realizate în anul anterior depunerii."
+      },
+      "q6": {
+        "question": "Cum pot redirecționa 3.5% din venit dacă am PFA sau am realizat venituri din chirii, drepturi de proprietate intelectuală sau venituri din străinătate?",
+        "answer": "Contribuabilii impusi in sistem real care au avut venituri din activități independente/agricole, drepturi de proprietate intelectuală și cedarea folosinței bunurilor și au obligația depunerii Declarației Unice pot allege destinația sumei de 3.5% din impozit completând secțiunea 6 a „Declarației unice privind impozitul pe venit si contributiile sociale datorate de persoanele fizice”."
+      }
+    }
   }
-}
+}
```

Comment:
The newline is missing at the end of this file.

Reply:
Done